doc/dev/developer_guide/index.rst

```diff
@@ -131,9 +131,15 @@ click on `New issue`_.
 Mailing list
 ------------
 
-Ceph development email discussions take place on the mailing list
-``ceph-devel@vger.kernel.org``. The list is open to all. Subscribe by
-sending a message to ``majordomo@vger.kernel.org`` with the line: ::
+The ``dev@ceph.io`` list is for discussion about the development of Ceph, 
+its interoperability with other technology, and the operations of the 
+project itself.  
+
+The ``ceph-devel@vger.kernel.org`` list is for discussion and patch review
+for the Linux kernel Ceph client component.
```

Comment:
I suppose add a note here as well that it used to be the all-encompassing dev list for future devs looking up archives.

Reply:
We should also keep the information on how to subscribe to this list too.